src/Inputs/InputPassword/InputPassword.tsx

```diff
@@ -1,7 +1,26 @@
-import { InputHTMLAttributes, forwardRef } from 'react'
-import StyledInput from '../styles'
+import { InputHTMLAttributes, forwardRef, useState } from 'react'
+import { StyledInput, StyledToggleInput } from '../styles'
+import { Icon } from '../../Icon'
 
-export const InputPassword = forwardRef<HTMLInputElement, InputHTMLAttributes<HTMLInputElement>>(
-  (props, ref) => <StyledInput type="password" ref={ref} {...props} />,
+export interface PasswordInputProps extends InputHTMLAttributes<HTMLInputElement> {
+  canToggle?: boolean;
```

Comment:
`canToggle` is a pretty ambitious name. Toggle what? It could be toggling anything.

`passwordToggle`, `revealPassword`, `canReveal`, `revealToggle`, `canRevealPassword`

Reply:
I had canTogglePassword originally, I thought you would not like it since it a bit verbose. :D I will switch it to `canRevealPassword`.

Reply:
Hahaha, `canRevealPassword` is great.